streaming/java/streaming-api/src/main/java/org/ray/streaming/python/descriptor/DescriptorPartition.java

```diff
@@ -0,0 +1,51 @@
+package org.ray.streaming.python.descriptor;
+
+import org.ray.streaming.api.partition.Partition;
+
+/**
+ * A DescriptorPartition is used to describe a python partition function.
+ * <p>
+ * Python worker can create a partition object based on information in this
+ * DescriptorPartition.
+ * <p>
+ * If DescriptorPartition is constructed from python serialized partition,
+ * python worker can directly deserialize serialized python partition to create
+ * python partition. If DescriptorPartition is constructed from moduleName and
+ * className/functionName, python worker will use `importlib` to load python
+ * partition function.
+ */
+public class DescriptorPartition implements Descriptor, Partition {
+  public static final DescriptorPartition BroadcastPartition = new DescriptorPartition(
```

Comment:
Is `BroadcastPartition` python side partition?
Why `DescriptorPartition(String moduleName, String className, String functionName) ` constructor is used for python?

Reply:
It's for `java` to specify a python partition. Java doesn't have a python partition object. It specify moduleName/className/functionName, so that we can use `importlib` to load partition in python code.